data/vss-core/README.md

```diff
@@ -3,26 +3,38 @@
 ## Introduction
 
 KUKSA.val is adapted to use Vehicle Signals Specification as defined by COVESA.
-The ambition is to always support the latest released version available at the [COVESA VSS release page](https://github.com/COVESA/vehicle_signal_specification/releases).
+The ambition is to always support the latest released version available at the
+[COVESA VSS release page](https://github.com/COVESA/vehicle_signal_specification/releases).
 In addition older versions may be supported. This folder contains copies of all versions supported.
 
 ## Supported VSS versions
 
+* [VSS 3.1.1](https://github.com/COVESA/vehicle_signal_specification/releases/tag/v3.1.1)
 * [VSS 3.0](https://github.com/COVESA/vehicle_signal_specification/releases/tag/v3.0)
```

Comment:
3.0 is listed as "supported" here, but below it says it is not. So either remove it here, or mention here only val-server